.github/workflows/prs_check-target-branch-label.yml

```diff
@@ -0,0 +1,36 @@
+name: Check for target branch labels
+
+on:
+  pull_request_target:
+    branches:
+      - 'v*.x'
+      - 'master'
+    types: ['labeled']
+  workflow_call:
+
+permissions: {}
+
+jobs:
+  detect_cherry_pick_target:
+    runs-on: ubuntu-latest
+    name: Check target branch labels
+    permissions:
+      issues: write
+      pull-requests: write
+      contents: write
+    if: ${{ github.event.label.name == 'needs cherry-pick' && github.event.pull_request.merged == false }}
```

Comment:
Woudln't this only run when the `needs cherry-pick` label is added, but not when the `v7.x` label is added?

Reply:
Yes... this is only to warn that we need a target label or the default approach will run